left_bus_name: enp3s0
right_bus_name: enp4s0
publishers:
  joint_state:
    topic: /joint_state
    queue_size: 1
    latch: false

  device_state:
    topic: /device_state
    queue_size: 1
    latch: false

  joint_trajectory_pub:
    topic: /joint_trajectory
    queue_size: 1
    latch: false

subscribers:
  my_subscriber_name:
    topic: /my_subscriber_topic
    queue_size: 1
    latch: false

  joint_trajectory:
    topic: /joint_trajectory
    queue_size: 1
    latch: false
